### ISSUE TITLE
508 Defect 2: Screen Reader, On Connected Aps page, "Have more questions" should be an h2.

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#28965]
Label: 

    - [508-defect-2]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2

Issue Title: 508 Defect 2: Screen Reader, On Connected Aps page, "Have more questions" should be an h2.
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a screen reader user, I expect headings to be nested properly.
<h2>Details</h2>
"Have more questions about connected aps" is currently an h3. Since it is not a subcatigory of the h2 representing the ap connected to the user account, it should be an h2.
